developer/docs/arvo/network.md

```diff
@@ -155,54 +175,60 @@ Sending a message to another urbit
 
-    ::  Allows one urbit to send the string 'Pong' to 
-    ::  another urbit.
-    ::
-    ::::  /===/app/pong/hoon
-      ::
-    !:
-    |%
-    ++  move  {bone card}
-    ++  card  $%  {$poke wire dock poke-contents}
-              ==
-    ++  poke-contents  $%  {$atom @}
-                       ==
-    --
-    |_  {bow/bowl $~}                                       ::<  stateless
-    ::
-    ++  poke-urbit
-      |=  to/ship
-      ^-  {(list move) _+>.$}
-      ~&  pong+'Outgoing pong!'
-      :_  +>.$
-      ~[[ost.bow %poke /sending [to dap.bow] %atom 'Pong']]
-    ::
-    ++  poke-atom
-      |=  tom/@
-      ^-  {(list move) _+>.$}
-      ~&  pong+'Incoming pong!'
-      ~&  pong+received+`@t`tom
-      [~ +>.$]
-    ::
-    ++  coup  |=(* [~ +>.$])
-    ::
-    --
+```
+::  Allows one urbit to send the string 'Pong' to       ::  1
+::  another urbit.                                      ::  2
+::                                                      ::  3
+::::  /===/app/pong/hoon                                ::  4
+  ::                                                    ::  5
+!:                                                      ::  6
+|%                                                      ::  7
++=  move  [bone card]                                   ::  8
++=  card  $%  [$poke wire dock poke-contents]           ::  9
+          ==                                            ::  10
++=  poke-contents  $%  [$atom @]                        ::  11
+                   ==                                   ::  12
+--                                                      ::  13
+|_  [bow/bowl $~]                                       ::  14
+::                                                      ::  15
+++  poke-urbit                                          ::  16
+  |=  to=ship                                           ::  17
+  ^-  [(list move) _+>.$]                               ::  18
+  ~&  pong+'Outgoing pong!'                             ::  19
+  :_  +>.$						::  20
+  ~[[ost.bow %poke /sending [to dap.bow] %atom 'Pong']] ::  21
+::							::  22
+++  poke-atom						::  23
+  |=  tom=@						::  24
+  ^-  [(list move) _+>.$]				::  25
+  ~&  pong+'Incoming pong!'				::  26
+  ~&  pong+received+`@t`tom				::  27
+  [~ +>.$]						::  28
+::							::  29
+++  coup  |=(* [~ +>.$])				::  30
+::							::  31
+--							::  32
+```
  
 Run it with these commands:
 
-    ~fintud-macrep:dojo> |start %pong
-    >=
-    ~fintud-macrep:dojo> :pong &urbit ~sampel-sipnym
-    >=
+```
+~fintud-macrep:dojo/examples> |start %pong
+>=
+~fintud-macrep:dojo/examples> :pong &urbit ~sampel-sipnym
+>=
+```
 
 Replace `~sampel-sipnym` with another urbit. The easiest thing to do is to start
 a comet, a free and disposable Urbit identity. If you don't know how to start a
-comet, see [the user setup section](/developer/using/setup/). Don't forget to start
+comet, see [the user setup section](/developer/docs/using/setup/). Don't forget to start
```

Comment:
Nope, the above path is current, but Galen and I don't like that this breaks things, so we'll probably change all of these links back in a different commit.